source/viz.md

```diff
@@ -1281,27 +1346,28 @@ Histogram of Michelson's speed of light data colored by experiment.
 ```{index} integer
 ```
 
-Alright great, {numref}`final_plot_colored` looks...wait a second! We are not able to distinguish 
-between different Experiments in the histogram! What is going on here? Well, if you 
+Alright great, {numref}`final_plot_colored` looks...wait a second! We are not able to distinguish
+between different Experiments in the histogram! What is going on here? Well, if you
 recall from Chapter {ref}`wrangling`, the *data type* you use for each variable
 can influence how Python and `altair` treats it. Here, we indeed have an issue
 with the data types in the `morley` data frame. In particular, the `Expt` column
 is currently an *integer*. But we want to treat it as a
-*category*, i.e., there should be one category per type of experiment.  
+*category*, i.e., there should be one category per type of experiment.
 
 ```{index} nominal, altair; :N
 ```
 
-To fix this issue we can convert the `Expt` variable into a `nominal`(categorical) type 
+To fix this issue we can convert the `Expt` variable into a `nominal`(categorical) type
```

Comment:
bad formatting with parens (needs a space before)

please check through whole chapter for this and fix